native/src/NavigatorContainer.ts

```diff
@@ -4,12 +4,12 @@ import { Dispatch } from 'redux'
 import Navigator from './Navigator'
 import { StoreActionType } from './redux/StoreActionType'
 
-type DispatchPropsType = {
+type DispatchProps = {
```

Comment:
According to our new rule, this should be called `NavigatorContainerProps`. But I find `DispatchProps` clearer. Thoughts?

Reply:
I think it should be `DispatchProps` since this is still legacy react-redux hoc style code. I would not put any effort in changing the naming here. DispatchProps for me is clearer and renaming it would only work as long as we don't have state props as well.

Reply:
ok